src/box/log.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#pragma once
+
+/**
+ * Check parameters for say_logger_init.
+ *
+ * @retval  0 - Parameters are valid.
+ * @retval -1 - Parameters are not valid (diag is set in this case).
+ */
+int
+say_check_cfg(const char *log,
+	      int level,
+	      int nonblock,
+	      const char *format);
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
```

Comment:
The `extern "C"` header is obviously misplaced.

Reply:
Dropped log.h file.